### ISSUE TITLE
Sound is not playing

### ISSUE THREAD

**Issue by valera5505** [malensek/3RVX#11]
I've just installed B4 and with default settings I can't hear sound on change.

**Comment by malensek** [comment]
I recently disabled this because it was causing performance problems, but it should be working again in B5:

http://matthew.malensek.net/projects/3RVX/3RVX3-B5.zip

Let me know how it works; I haven't tested the sound effects a whole lot at this point. Thanks!

**Comment by valera5505** [comment]
Ok, now it works! But I think, one volume up key press should increase volume by one square (in v2 I have percentage setting, but it doesn't work)

**Comment by malensek** [comment]
Yeah, let me add this to the list of things to do for B6. The increment is kind of messed up right now.

**Comment by malensek** [comment]
Hey @valera5505, were you able to test the latest beta? I think the behavior you were talking about should be fixed now.

I'll also get a custom increment setting implemented.

**Comment by valera5505** [comment]
Keyboard hotkeys work well, but mouse buttons work as usual.

**Comment by malensek** [comment]
How about with beta 8? I made a couple more changes to how the sound effects are handled.

**Comment by valera5505** [comment]
Now everything works as expected.

**Comment by valera5505** [comment]
In Task Manager the problem remaind

**Comment by valera5505** [comment]
Aslo in some programs (I think it's because they launch with administrator privileges)

**Comment by malensek** [comment]
If you run 3RVX as Administrator, does it work for all programs?

**Comment by malensek** [comment]
It looks like this can be worked around by getting a certificate and signing the app for UI access... I'll look into what options are available.
